### PR TITLE
Increased accuracy by using Different Algorithms

### PR DESCRIPTION
**Pull request by keshav-bagaria**
Hi,

I have added a submission folder. And within the folder I have submitted my ipynb notebook.

I have added XgBoost and LightGbm classifier implementations which have increased the accuracy close to 99%.

### CODE REVIEW

**Review by sarthakpant772 — APPROVED** [review]
Good work 👍🏼
